pkg/jailer/jailer.go

```diff
@@ -81,3 +83,20 @@ func GetUserCred() (*syscall.Credential, error) {
 
 	return &syscall.Credential{Uid: uid, Gid: gid}, nil
 }
+
+func WhitelistEnvvars(envvars []string) []string {
+	wl := settings.WhitelistEnvironmentVars.Get()
+	envWhiteList := strings.Split(wl, ",")
+
+	if len(envWhiteList) == 0 {
+		return envvars
+	}
+
+	for _, wlVar := range envWhiteList {
```

Comment:
Do you think we should do `trimpSpace` on the each wlVar?

Reply:
Also do you know if Getenv is case sensitive? And can these variables be configured in a lower case on users' setups

Reply:
Added trim space. Per discussion not going to do anything about case, user can utilize the setting if they need to change the case of the envvar being looked up.